Library/Homebrew/cmd/pull.rb

```diff
@@ -179,9 +193,9 @@ def pull
             package = Bintray.package f.name
             version = f.pkg_version
             curl "-w", '\n', "--silent", "--fail",
-              "-u#{bintray_user}:#{bintray_key}", "-X", "POST",
-              "-d", '{"publish_wait_for_secs": -1}',
-              "https://api.bintray.com/content/homebrew/#{repo}/#{package}/#{version}/publish"
+                 "-u#{bintray_user}:#{bintray_key}", "-X", "POST",
+                 "-d", '{"publish_wait_for_secs": -1}',
+                 "https://api.bintray.com/content/homebrew/#{repo}/#{package}/#{version}/publish"
```

Comment:
Fair objection. This and the previous one were a result of RubyMine's autoformatting. I thought I had fixed that by changing the "continuation indent" settings, but it didn't take for these. I'll revert.